CHANGELOG.md

```diff
@@ -4,6 +4,36 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.0.0-rc3 (2018-10-18)
+
+This release is our third 1.0 release candidate. We expect to only merge bug fixes between now and our final 1.0 release. This is a big milestone for us and signifies our maturity and enterprise readiness.
+
+**PLEASE NOTE** that release candidate (rc) builds will only be made available via GitHub and Docker, and _not_ on other release channels. Please help us test these release candidates out if you can!
+
+**Potential breaking change**: We updated our error codes to be prefixed with `TS` instead of the old `IO` prefix. If you were checking for these error codes by name, please update your checks.
```

Comment:
This was in a previous rc release, although I'm not sure which. If we want to keep propagating this warning, maybe say which version it showed up in?

Reply:
It was rc2, I kept it in case someone is coming to the RCs for the first time on this one.